Sources/TuistLoader/Loaders/GeneratorModelLoader.swift

```diff
@@ -83,6 +83,10 @@ public class GeneratorModelLoader: GeneratorModelLoading {
         let xcodeFileName = xcodeFileNameOverride(from: config, for: model)
         enrichedModel = enrichedModel.replacing(fileName: xcodeFileName)
 
+        // Xcode project organization name
+        let organizationName = organizationNameOverride(from: config)
+        enrichedModel = enrichedModel.replacing(organizationName: organizationName)
```

Comment:
I believe an override needs to take place only if it exists, otherwise for projects that don't specify it via Config will never be able to set it via Project.swift (it always gets replaced with `nil`) 😅

```suggestion
        if let organizationName = organizationNameOverride(from: config) {
            enrichedModel = enrichedModel.replacing(organizationName: organizationName)
         }
```

Reply:
doh! I couldn't find a unit test that would be good to validate this use case. Where would be the best place for this?

Reply:
Also since I copied the logic from xcodeFileName, wouldn't the problem occurs with that too?

Reply:
I believe `xcodeFileName` isn't exposed via `Project.swit` hence why this was perhaps unnoticed before (the only public API is via`Config.swift`).

Reply:
>  I couldn't find a unit test that would be good to validate this use case. Where would be the best place for this?

The `GeneratorModelLoaderTests` would be a good place for this, I believe you already [added a test there](https://github.com/tuist/tuist/pull/1062/files#diff-fd234ee020fbc9befcf44b8609f18638R204) - its worth another test cases where there is no config to ensure the name specified in the manifest is respected :)